packages/cactus-odap-hermes/src/main/json/openapi.json

```diff
@@ -0,0 +1,1216 @@
+{
+    "openapi": "3.0.3",
+    "info": {
+        "title": "Hyperledger Cactus Plugin - Connector Besu",
+        "description": "Can perform basic tasks on a Besu ledger",
+        "version": "0.0.1",
+        "license": {
+            "name": "Apache 2.0",
+            "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+        }
+    },
+    "servers": [
+        {
+            "url": "https://www.cactus.stream/{basePath}",
+            "description": "Public test instance",
+            "variables": {
+                "basePath": {
+                    "default": ""
+                }
+            }
+        },
+        {
+            "url": "http://localhost:4000/{basePath}",
+            "description": "    ",
+            "variables": {
+                "basePath": {
+                    "default": ""
+                }
+            }
+        }
+    ],
+    "components": {
+        "schemas": {
+            "CredentialProfile":{
+                "type":"string",
+                "enum": [
+                    "SAML",
+                    "OAUTH",
+                    "X509"
+                ]
+            },  
+            "PayloadProfile":{
+                "type":"object",
+                "properties": {
+                    "assetProfile":{
+                        "$ref":"#/components/schemas/AssetProfile"
+                    },
+                    "capabilities":{
+                        "type":"string"
+                    }
+                },
+                "required": [
+                    "assetProfile"
+                ]
+            },
+            "ApplicationProfile":{
+                "type":"object"
+            },
+            "Payload":{
+                "type":"object"
+            },
+            "AssetProfile":{
+                "type":"object",
+                "properties": {
+                    "Issuer":{
```

Comment:
@jscode017 @RafaelAPB Can the casings of the property names be `camleCase` instead of `CapitalizedCase` or is this strictly mandated by the standard ?

Reply:
ok ,resolve with new commit